web/app/mu-plugins/pantheon-loader.php

```diff
@@ -1,97 +0,0 @@
-<?php
```

Comment:
Did some additional testing here (since creating CMS-837) and learned that `roots/bedrock-autoloader` [loads](https://github.com/roots/bedrock-autoloader/blob/master/src/Autoloader.php#L57) any plugin added to the `mu-plugins` directory. It also (like this file did) [displays](https://github.com/roots/bedrock-autoloader/blob/master/src/Autoloader.php#L77) the autoloaded plugins in the WP admin.

Reply:
I'm glad you tested that. I remember seeing something in Bedrock docs that said something about that but I hadn't gotten around to validating or testing it.